redisinsight/api/src/modules/browser/keys/scanner/strategies/standalone.scanner.strategy.ts

```diff
@@ -162,19 +162,30 @@ export class StandaloneScannerStrategy extends ScannerStrategy {
     client: RedisClient,
     keys: RedisString[],
     filterType?: RedisDataType,
+    getSize?: boolean,
+    getTtl?: boolean,
```

Comment:
I'm wondering if changing the signature to object would be too much of an effort